services/manage/app/templates/manage/orders/edit/lineitems/edit/ads/edit.hbs

```diff
@@ -37,13 +37,20 @@
       </div>
 
       <div class="row">
-        <div class="col">
+        <div class="col-md-9">
           {{ad/-fields/url
             form=form
             value=model.url
             on-change=(action "setUrl")
           }}
         </div>
+        <div class="col-md-3">
+          {{lineitem/-fields/paused
```

Comment:
This shouldn't use the lineitem paused field and instead should have an ad paused field component